source/manual/common-aws-tasks-for-2nd-line-support.html.md

```diff
@@ -247,18 +121,18 @@ decisions have been made.
 
 ## How are instances provisioned?
 
-We typically use Terraform config in [govuk-aws][] to provision infrastructure,
+We typically use Terraform config in [govuk-aws] to provision infrastructure,
 such as EC2 instances.
 
 There are a few exceptions to this, such as ad-hoc instances started from
 Concourse via the AWS CLI - these are mainly for data science projects.
 
-We use [userdata scripts][] to run commands on our instances at launch. These
+We use [userdata scripts] to run commands on our instances at launch. These
 scripts install various core bits of software needed by a particular instance
-and then typically use [govuk-puppet][] to provision our instances.
+and then typically use [govuk-puppet] to provision our instances.
```

Comment:
Should this bit about govuk-puppet be removed?

Reply:
I thought about removing this whole section about EC2 instances but thought I'd leave it in for the moment because we still have a few (like crawler for instance, and the Puppet-based stuff that supports it). Hopefully soon we'll be able to delete all this though!